java/arcs/android/devtools/IDevToolsService.aidl

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.devtools;
+
+
+/**
+ * Exposed API for the DevToolsService.
+ *
+ * TODO: subject to change.
+ */
+interface IDevToolsService {
+    /** Sends the [str] to the remote device */
+    oneway void send(String str);
+
+    /** Start the [DevWebSocket]. */
+    void start();
+
+    /** Cancel any ongoing actions. */
+    void cancel();
```

Comment:
I don't know if this needs to be provided by the AIDL interface, unless you foresee situations where the client would need to cancel (typically they'd just call `context.unbindService(serviceConnection)`).

That's why I was saying that it'd be best to manage the scope at the Service, (including canceling it in onDestroy).

I'm not sure I follow why the crashes you mentioned were happening?

Reply:
It was saying something about leaking the service connection. I went back to having the scope in the DevToolsService and everything worked this time. I must have had some other unintended change. 🤷‍♀️  Either way, it's back to the way it should be now.